transport-native-epoll/src/main/c/io_netty_channel_unix_FileDescriptor.c

```diff
@@ -24,3 +25,16 @@ JNIEXPORT int JNICALL Java_io_netty_channel_unix_FileDescriptor_close(JNIEnv* en
    }
    return 0;
 }
+
+JNIEXPORT int JNICALL Java_io_netty_channel_unix_FileDescriptor_open(JNIEnv* env, jclass clazz, jstring path) {
+
+    const char* f_path = (*env)->GetStringUTFChars(env, path, 0);
+
+    int res = open(f_path, O_WRONLY | O_CREAT | O_TRUNC, 0644);
```

Comment:
http://man7.org/linux/man-pages/man2/open.2.html

> mode specifies the permissions to use in case a new file is
              created.  This argument must be supplied when O_CREAT or
              O_TMPFILE is specified in flags; if neither O_CREAT nor
              O_TMPFILE is specified, then mode is ignored.  The effective
              permissions are modified by the process's umask in the usual
              way: The permissions of the created file are (mode & ~umask).
              Note that this mode applies only to future accesses of the
              newly created file; the open() call that creates a read-only
              file may well return a read/write file descriptor.

666 seems fine to me.